src/OpenFeature.Providers.Ofrep/Configuration/OfrepConfiguration.cs

```diff
@@ -0,0 +1,54 @@
+using System.Text.Json.Serialization;
+
+namespace OpenFeature.Providers.Ofrep.Configuration;
+
+/// <summary>
+/// Configuration options for the OFREP provider.
+/// </summary>
+public class OfrepConfiguration
+{
+    /// <summary>
+    /// Gets or sets the base URL for the OFREP API.
+    /// </summary>
+    [JsonPropertyName("baseUrl")]
+    public string BaseUrl { get; set; }
+
+    /// <summary>
+    /// Gets or sets the timeout for HTTP requests. Default is 10 seconds.
+    /// </summary>
+    [JsonPropertyName("timeout")]
+    public TimeSpan Timeout { get; set; } = TimeSpan.FromSeconds(10);
+
+    /// <summary>
+    /// Gets or sets additional HTTP headers to include in requests.
+    /// </summary>
+    [JsonPropertyName("headers")]
+    public Dictionary<string, string>? Headers { get; set; }
+
+    /// <summary>
+    /// Gets or sets the authorization header value.
+    /// </summary>
+    [JsonPropertyName("authorizationHeader")]
+    public string? AuthorizationHeader { get; set; }
```

Comment:
Do we need those 2 options?
A single headers option is enough if we want to set an authorization header you can do it directly in headers.

This is how it works for both the GO and JS provider.